### PR TITLE
Bump to jQuery 2

### PR DESCRIPTION
**Pull request by ghost**
Since no blocking point was raised in #6557, I propose we switch to jQuery 2.

### CODE REVIEW

File: spec/javascripts/mobile/mobile_comments_spec.js [hunk 1]
```diff
@@ -82,14 +82,8 @@ describe("Diaspora.Mobile.Comments", function(){
   describe("createComment", function () {
     beforeEach(function() {
       spec.loadFixture("aspects_index_mobile_post_with_comments");
-      var commentBoxHtml = spec.fixtureHtml("comments_mobile_commentbox");
       var link = $(".stream .comment-action").first();
       Diaspora.Mobile.Comments.showCommentBox(link);
-      jasmine.Ajax.requests.mostRecent().respondWith({
```
Comment (denschub) [inline spec/javascripts/mobile/mobile_comments_spec.js]: Why did you remove that test? Is loading the recent comments covered somewhere else?

Comment (AugierLe42e) [inline spec/javascripts/mobile/mobile_comments_spec.js]: It's not a test, it's enclosed in the `beforeEach()` function. This line [is buggy](https://travis-ci.org/diaspora/diaspora/jobs/150391203#L813) with jQuery 2 and since it is not relevant for the test, I thought it was not a big deal to remove it.

Comment (denschub) [inline spec/javascripts/mobile/mobile_comments_spec.js]: True, it's a shim for the most recent comments ajax request. I'm surprised we don't break any tests by removing the shim. @svbergerem?

Comment (AugierLe42e) [inline spec/javascripts/mobile/mobile_comments_spec.js]: In this case, no ajax request should be performed during the test. So I'm actually not quite sure to which ajax call this line was supposed to answer.

Comment (svbergerem) [inline spec/javascripts/mobile/mobile_comments_spec.js]: This has been added in #6543 but I also don't see why. We spy on `jQuery.ajax` and make sure that it hasn't been called so this change looks good to me.

Comment (denschub) [inline spec/javascripts/mobile/mobile_comments_spec.js]: Thanks

Comment (svbergerem) [inline spec/javascripts/mobile/mobile_comments_spec.js]: @AugierLe42e You might also want to remove https://github.com/diaspora/diaspora/blob/develop/spec/controllers/jasmine_fixtures/comments_spec.rb then.

Comment (AugierLe42e) [inline spec/javascripts/mobile/mobile_comments_spec.js]: Sure, yes. Thx ;)

**Comment by svbergerem** [comment]
Thank you!
